components/Modals/CreateItemModal.tsx

```diff
@@ -218,18 +221,55 @@ export const CreateItemModal: React.FC<CreateItemModalProps> = ({
           return;
         }
 
-        const claimable = pad('0x00');
-
-        //TODO: the claimable addresses still need added to requirements
-
-        // if (claimableToggle) {
-        //   const addresses = whitelistedClaimers.split(',');
-        //   const trimmedAddresses = addresses.map(address => address.trim());
-
-        //   if (trimmedAddresses.length === 0) {
-        //     claimable = pad('0x01');
-        //   }
-        // }
+        let claimable = pad('0x00');
+
+        if (claimableToggle) {
+          const addresses = whitelistedClaimers.split(',');
+          const trimmedAddresses = addresses.map(address => address.trim());
+
+          if (trimmedAddresses.length === 0) {
+            claimable = pad('0x01');
+          } else {
+            const leaves: ClaimableItemLeaf[] = trimmedAddresses.map(
+              address => {
+                return [
+                  BigInt(game.items.length),
+                  getAddress(address),
+                  BigInt(itemSupply),
```

Comment:
We need to update this to set only the amount we want the user to be able to claim.